features/securebackup/impl/src/main/kotlin/io/element/android/features/securebackup/impl/enter/SecureBackupEnterRecoveryKeyView.kt

```diff
@@ -91,6 +94,11 @@ private fun ColumnScope.Buttons(
             state.eventSink.invoke(SecureBackupEnterRecoveryKeyEvents.Submit)
         }
     )
+    TextButton(
+        text = stringResource(id = R.string.screen_recovery_key_confirm_lost_recovery_key),
+        modifier = Modifier.fillMaxWidth(),
+        onClick = onCreateRecoveryKey,
+    )
```

Comment:
I think this button should be disabled when `state.submitAction.isLoading()`. WDYT?

Made this remark when seeing this screenshot: https://github.com/element-hq/element-x-android/pull/2745/files#diff-dbd46661ff7c38b3aefb7d016c5a6e54348191cc5ac6a6f1bd71213ad7db2019

Reply:
Actually, when the progress indicator is displayed the button is not clickable, so it's 'disabled' although it doesn't display the disabled state. It's what we have everywhere else I believe.

Reply:
Yes, for "Continue" button, but I am talking about the other one "Lost your recovery key?". Sorry if it was not clear.

Reply:
So to clarify:
The two buttons must be disabled during the submit action loading, to avoid user navigating away during the process.

Adding at line 101:
```kotlin
enabled = !state.submitAction.isLoading()
```
should be fine.